deploy-config/production.yml

```
@@ -1,8 +1,10 @@
 env: production
 instances: 2
 memory: 1.5G
+command: gunicorn -c /home/vcap/app/gunicorn_config.py application
```

Same note here about the command that production should have:

```suggestion
command: newrelic-admin run-program gunicorn -c /home/vcap/app/gunicorn_config.py application
```